eslint-local-rules/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    /** This rule is used to prevent the use of `React` namespace. ex: `React.useEffect` -> `useEffect` */
+    'no-react-namespace': {
+        create(context) {
+            return {
+                MemberExpression(node) {
+                    if (node.object.type === 'Identifier' && node.object.name === 'React') {
+                        context.report({
+                            node,
+                            message: `Use ${node.property.name} instead of React.${node.property.name}.`,
+                            fix: fixer => {
```

Comment:
Later need to update this fixer to also update the import statement.

cc: @thisyahlen-deriv 🙏🏻